src/sys/aio.rs

```diff
@@ -231,6 +617,46 @@ impl<'a> AioCb<'a> {
     }
 
     /// Cancels an outstanding AIO request.
+    ///
+    /// The operating system is not required to implement cancellation for all
+    /// file and device types.  Even if it does, there is no guarantee that the
+    /// operation has not already completed.  So the caller must check the
+    /// result and handle operations that were not canceled or that have already
+    /// completed.
+    ///
+    /// See also
```

Comment:
I would rephrase to indicate this is an external link like "See the man pages for `aio_cancel` for more details." This applies for all the uses of "See also LINK".